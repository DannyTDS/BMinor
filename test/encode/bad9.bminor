"Hello World!\0xF"