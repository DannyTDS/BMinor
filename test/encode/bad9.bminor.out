Error: input string not beginning with a double quote.
Cannot decode string .
